website/docs/r/nsxt_app_port_profile.html.markdown

```diff
@@ -22,25 +22,28 @@ NSX-T Data Center, you can create custom Application Port Profiles.
 
 ```hcl
 resource "vcd_nsxt_app_port_profile" "icmpv4" {
-  org = "System"
```

Comment:
we should leave this line to explicitly show what type or ORG is needed

Reply:
it is fixed.